app/assets/javascripts/components/student/student_contact.es6.jsx

```diff
@@ -30,17 +30,18 @@ class StudentContact extends Component {
     var student = this.props.student;
     return (
         <div style={this.styles.container}>
-          <h2 style={this.styles.heading}>{'Contact'}</h2>
-            {student.phone_number}
-            {student.home_address}
-            {student.email}
-          <h2 style={this.styles.heading}>{'School'}</h2>
+          <h4>{'Contact'}</h4>
+            <h6>{student.cell_phone}</h6>
+            <h6>{student.home_phone}</h6>
+            <h6>{student.email}</h6>
+            <h6>{student.home_address}</h6>
+          <br/>
```

Comment:
I don't think you need this `<br/>` tag, since heading tags by default start a new line after themselves!